### PR TITLE
Docker stats will wait, with no output when called on a non-running container

### PR DESCRIPTION
**Pull request by SvenDowideit**
@crosbymichael I presume this was intentional?

Signed-off-by: Sven Dowideit SvenDowideit@home.org.au

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@fredlf @jamtur01

**Comment by LK4D4** [comment]
Maybe we should prepopulate table with zeroes always first?

**Comment by crosbymichael** [comment]
@LK4D4 prepopulate should be a better idea.

File: docs/sources/reference/api/docker_remote_api_v1.17.md [hunk 1]
```diff
@@ -521,7 +521,9 @@ Status Codes:
 
 `GET /containers/(id)/stats`
 
-This endpoint returns a live stream of a container's resource usage statistics.
+This endpoint will returns a live stream of a running container's resource usage statistics.
```
Comment (fredlf) [inline docs/sources/reference/api/docker_remote_api_v1.17.md]: Agreement error. Delete "will"

File: docs/sources/reference/api/docker_remote_api_v1.17.md [hunk 2]
```diff
@@ -521,7 +521,9 @@ Status Codes:
...
+It will attach to any created container, but will only send resource usage information
+when the container is running.
```
Comment (fredlf) [inline docs/sources/reference/api/docker_remote_api_v1.17.md]: Eliminate conditional, make active voice: It attaches to any created container, but only sends resource usage information when the container is running.

File: docs/sources/reference/commandline/cli.md [hunk 3]
```diff
@@ -2012,6 +2012,11 @@ more details on finding shared images from the command line.
 
       --help=false       Print usage
 
+The `docker stats` command will listen for a stats stream for all created containers
+requested. It will however only display data for containers when they are running.
```
Comment (fredlf) [inline docs/sources/reference/commandline/cli.md]: Simplify/clarify:

The `docker stats` command listens for a stats stream for all the requested created containers. However, data is only displayed for running containers.

**Pull request closed without merging**